tap_google_analytics/client.py

```diff
@@ -125,9 +149,10 @@ def _make_request(self, method, url, params=None, data=None):
         else:
             response = self.session.request(method, url, headers=headers, params=params)
 
-        error_message = _is_json(response) and response.json().get("error", {}).get("message")
-        if response.status_code == 400 and error_message:
-            raise Exception("400 Client Error: Bad Request, details: {}".format(error_message))
+        response_is_json = _is_json(response)
+        error_message = response_is_json and response.json().get("error", {}).get("message")
```

Comment:
So because this is an `and` does that mean it won't try to call the second part of this if the response is not a json? Just checking...

Reply:
Correct, boolean operators are short-circuiting in python.